<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un document</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: white;
}

QLabel {
    font-size: 11pt;
    color: #333333;
}

QRadioButton {
    font-size: 10pt;
    color: #333333;
    spacing: 8px;
}

QPushButton {
    font-size: 10pt;
    padding: 6px 20px;
    border-radius: 3px;
}

QPushButton#okButton {
    background-color: #0078D4;
    color: white;
    border: 1px solid #0078D4;
}

QPushButton#okButton:hover {
    background-color: #106EBE;
}

QPushButton#cancelButton {
    background-color: white;
    color: #333333;
    border: 1px solid #CCCCCC;
}

QPushButton#cancelButton:hover {
    background-color: #F0F0F0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Quel type de document voulez-vous ajouter ?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QRadioButton" name="radioDevis">
     <property name="text">
      <string>Devis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioBonCommande">
     <property name="text">
      <string>Bon de commande</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioBonLivraison">
     <property name="text">
      <string>Bon de livraison</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioFacture">
     <property name="text">
      <string>Facture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioFactureAvoir">
     <property name="text">
      <string>Facture d'avoir</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
